pkg/cmd/attestation/io/handler.go

```diff
@@ -59,3 +60,19 @@ func (h *Handler) VerbosePrintf(f string, v ...interface{}) (int, error) {
 
 	return fmt.Fprintf(h.IO.ErrOut, f, v...)
 }
+
+func (h *Handler) PrintTable(headers []string, rows [][]string) error {
+	t := tableprinter.New(h.IO, tableprinter.WithHeader(headers...))
```

Comment:
Minor: I wish `internal/tableprinter/table_printer.go` didn't use `cs.LightGrayUnderline` for headers! I must be the only person with a white termial.

<img width="980" alt="Screenshot 2024-04-24 at 3 56 45 PM" src="https://github.com/cli/cli/assets/167759/dc5d98de-632e-41ab-a500-b1338a9671fe">

Reply:
Good to know. Maybe we can look into this in the future.

Reply:
Yes indeed I had this concern originally when this was PRed grrr. Definitely something to address!